Makefile

```diff
@@ -124,7 +124,8 @@ verify: tidy download ## Verify code. Includes dependencies, linting, formatting
 	cp  $(KARPENTER_CORE_DIR)/pkg/apis/crds/* pkg/apis/crds
 	hack/validation/requirements.sh
 	hack/validation/labels.sh
-	$(foreach dir,$(MOD_DIRS),cd $(dir) && golangci-lint run $(newline))
+	MOD_DIRS=$(shell echo $(MOD_DIRS) | sed 's/.\/website//g')
```

Comment:
I feel like we might be able to exclude the website in _all_ cases. When the `go.mod` was included, even in the `go mod tidy` step it was causing issues since `go` doesn't see the usage of the packages so it cleans up the `go.mod` to be empty every time

```makefile
MOD_DIRS = $(shell find . -path "./website" -prune -o -name go.mod -type f -print | xargs dirname
```